lua/telescope/actions/init.lua

```diff
@@ -122,6 +122,7 @@ function actions._goto_file_selection(prompt_bufnr, command)
         end
       end
     end
+    vim.api.nvim_command("doautocmd filetypedetect BufRead " .. vim.fn.fnameescape(filename))
```

Comment:
This should probably not be here, but in the `filename` part of the else block above. `bufnr` means we already have a buffer for it I think (which means we would call this stuff again, which would be weird I think)

Reply:
What about unloaded buffers? If we have a old session? With `show_all_buffers` we will still display them and are able to change to them